phys/module_diag_trad_fields.F

```diff
@@ -175,7 +173,6 @@ SUBROUTINE trad_fields ( u,v,w,t,qv,zp,zb,pp,pb,p,pw,            &
 
                ! Surface Pressure
```

Comment:
@smileMchen 
Delete the section for psfc, not just that one line

Reply:
@davegill 
The extra lines are deleted.